src/torrent_handle.cpp

```diff
@@ -265,6 +265,29 @@ namespace libtorrent {
 		async_call(&torrent::pause, bool(flags & graceful_pause));
 	}
 
+	boost::uint64_t torrent_handle::flags() const
+	{
+		return sync_call_ret<boost::uint64_t>(0, &torrent::flags);
+	}
+
+	void torrent_handle::set_flags(boost::uint64_t flags, boost::uint64_t mask) const
+	{
+		void (torrent::*set_flags)(boost::uint64_t, boost::uint64_t) = &torrent::set_flags;
```

Comment:
in fact, ``unset_flags()`` could call that same function too, but just forward the flags as the mask